stdlib/2.7/__builtin__.pyi

```diff
@@ -539,9 +539,9 @@ class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     def popitem(self) -> Tuple[_KT, _VT]: ...
     def setdefault(self, k: _KT, default: _VT = ...) -> _VT: ...
     @overload
-    def update(self, m: Mapping[_KT, _VT]) -> None: ...
+    def update(self, m: Mapping[_KT, _VT], **kwargs: Mapping[_KT, _VT]) -> None: ...
```

Comment:
The type of `**kwargs` is the type of the dictionary values instead of the entire mapping/dictionary. In this case it would be just `_VT` (key would be implicitly `str`).